course/templates/course/course_base.html

```diff
@@ -26,7 +26,7 @@
 {% block content %}
 <div data-taggings="{{ get_taggings|join:' ' }}"  class="row">
     <div class="col-sm-2 hidden-xs">
-        <nav class="course-menu" id="main-course-menu" aria-labelledby="course-menu-heading">
+        <nav class="course-menu" id="main-course-menu" aria-labelledby="course-menu-heading"></nav>
```

Comment:
You weren't supposed to close the nav element on this line.

Reply:
The IDE doing autocorrections 🤦🏽‍♂️